frontend/libs/studio-components/src/components/StudioPageError/StudioPageError.mdx

```diff
@@ -0,0 +1,22 @@
+import { Meta, Canvas, Story } from '@storybook/blocks';
+import { Heading, Paragraph } from '@digdir/designsystemet-react';
+import * as StudioPageErrorStories from './StudioPageError.stories';
+
+<Meta of={StudioPageErrorStories} />
+
+<Heading level={1} size='small'>
+  StudioPageError
+</Heading>
+<Paragraph>
+  The `StudioPageError` component is used to display error messages on a page. It includes an alert
+  box with a title and a message.
+</Paragraph>
+
+<Heading level={2} size='small'>
+  Props
+</Heading>
+<Paragraph>`StudioPageError` component has tow props:</Paragraph>
+<Paragraph>- `title`: A string representing the title of the error message.</Paragraph>
+<Paragraph>- `message`: A string representing the error message content.</Paragraph>
```

Comment:
Siden dette er en liste, bør vi bruke `List` fremfor `Paragraph`. Men i dette tilfellet er nok `Table` enda bedre egnet:
```suggestion
<Table size="sm">
  <Table.Head>
    <Table.Row>
      <Table.HeaderCell>Name</Table.HeaderCell>
      <Table.HeaderCell>Description</Table.HeaderCell>
    </Table.Row>
  </Table.Head>
  <Table.Body>
    <Table.Row>
      <Table.Cell>
        `title`
      </Table.Cell>
      <Table.Cell>
        A string representing the title of the error message.
      </Table.Cell>
    </Table.Row>
    <Table.Row>
      <Table.Cell>
        `message`
      </Table.Cell>
      <Table.Cell>
        A string representing the error message content.
      </Table.Cell>
    </Table.Row>
  </Table.Body>
</Table>
```
For øvrig er jeg usikker på om vi trenger disse `mdx`-filene i det hele tatt. De skaper bare ekstra vedlikeholdsbehov. Men du kan la det være nå, så tar jeg det opp når alle er tilbake på jobb.

Reply:
fikset